recipes/telescope/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "telescope" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [py27]
+  script: "python setup.py install --single-version-externally-managed --record=record.txt"
+
+about:
+  home: 'https://github.com/mlbendall/telescope'
+  license: MIT
+  summary: Single locus resolution of Transposable ELEment expression.
+
+source: 
+  url: https://github.com/mlbendall/telescope/archive/v1.0.3.tar.gz
+  sha256: 115928f5aa347bf2416a15e9d790744e42553bb1118809b5677788e649269f74
+
+requirements:
+  build:
```

Comment:
With the comment from above, the `build` section under requirements should be `host`. `build` is usually used when you have compilers.